interfaces/builtin/common.go

```diff
@@ -55,6 +55,9 @@ type commonInterface struct {
 	connectedSlotKModModules []string
 	permanentPlugKModModules []string
 	permanentSlotKModModules []string
+
+	usesPtraceTrace     bool
```

Comment:
If you follow my naming suggestion above this could become
```
usePtraceTrace bool
suppressPtraceTrace bool
```

Reply:
I chose 'usesPtraceTrace' since the interface is declaring it uses it. I chose 'suppressPtraceTrace' because the interface is saying "please supress the ptrace trace denials if you can". I can rename things if you still want.